configure

```diff
@@ -333,6 +333,11 @@ parser.add_option('--enable-static',
     dest='enable_static',
     help='build as static library')
 
+parser.add_option('--enable-asan',
+    action='store_true',
+    dest='enable_asan',
+    help='build with asan')
```

Comment:
Can you put it before --enable-static?  I'd like to keep a semblance of alphabetical order.